Update python example

saisk8 opened the pull request:
The `ensureDirExists` has been updated to use the built-in python method. Also, please note that the definition now always returns `True`.

saisk8 committed:
Use built-in method for nested dir creation

saiki-k commented on examples/python/utils.py:
```diff
@@ -17,8 +17,5 @@ def generateRandomUUID():
-        return True
-    ensureDirExists(dirName)
-    os.mkdir(dirName)
+    os.makedirs(filePath, exist_ok=True)
```
We can completely do away with ensureDirExists fn in this case.

Can you please send an updated PR; removing ensureDirExists from utils.py, and its import in main.py; and replacing the ensureDirExists call(s) in main.py, with os.makedirs call(s)?

saisk8 replied:
That makes sense, I have updated the PR.

saisk8 committed:
Remove ensureDirExists function

This function is being removed in favor of the python's built-in `makedirs` method of the os package.

saisk8 committed:
Remove unused methods

saisk8 committed:
Remove newline
